merlin/systems/dag/ops/faiss.py

```diff
@@ -82,6 +163,33 @@ def compute_input_schema(
         deps_schema: Schema,
         selector: ColumnSelector,
     ) -> Schema:
+        """
+        Compute the input schema of this node given the root, parents and dependencies schemas of
```

Comment:
sugg: "...root, parents, and dependencies..." (serial commas are typically preferred.)

Please add ending punctuation to the parameter descriptions.